sql/modules/Assets.sql

```diff
@@ -843,6 +885,12 @@ AS $$
     SELECT * FROM account__get_by_link_desc('asset_loss');
 $$ language sql;
 
+CREATE OR REPLACE FUNCTION asset_report__get_cash_accts()
```

Comment:
1. Should exclude obsolete account.
2. Instead of getting all accounts, better to filter only `AR_paid`, `AP_paid`?
Is there any other use case which need all accounts?
(Because, we used only those account when we manually add cash account in Journal line)

Reply:
Thanks! The obsolete account fix was added.

The lack of selection of accounts based on "AR_paid" or "AP_paid" was intentional: these are intended to select the drop-down values on payments from vendors and to customers. However, for fixed assets you may want to use other accounts (e.g. suspense accounts) which don't apply to payments/receipts. When I created this,  I estimated that the added complexity of yet another "knob" (checkbox on the account configuration screen) is not warranted given the fact that fixed assets disposal will be far less common than processing payments -- meaning that added flexibility of the selection of the right account in the drop-down outweighs the productivity lost by needing to scroll to the right account. Agreed?